README.md

```diff
@@ -2,15 +2,30 @@
 
 ## Usage
 
-To run in a docker container clone this repo then run:
-
-### Concatenating folder structure files in one file
+The `buildForms` script can be used to bundle all form configurations and generate a folder structure for each supported project.
 
+```sh
```

Comment:
I know it's a given, but maybe document that you have to do `npm install` first. I think the bash script from before did that for you.